doc/sphinx/source/index.txt

```diff
@@ -141,6 +141,8 @@ For current issues and open feature requests please look through the
    fep
    utilities
    forcefields
+   ensemble
+   analysis
```

Comment:
ensemble should be inside analysis

Reply:
@orbeckst think I got the docs updated to what you wanted.